package.json

```diff
@@ -48,7 +49,7 @@
     "husky": "6.0.0",
     "jest-dom": "^4.0.0",
     "lint-staged": "^13.1.0",
-    "performance-now": "^2.1.0"
+    "jest-dom": "^4.0.0"
```

Comment:
no code change for adding jest-dom?

Reply:
@ps48 can you remove the extra dep? I just checked the original PR doesn't have this might have happened during merge

Reply:
The duplication arose due to auto-merge in cherry pick. Fixed it.